app/components/article-featured-video.js

```diff
@@ -28,20 +28,15 @@ export default Component.extend(RenderComponentMixin, {
 	autoplayCookieName: 'featuredVideoAutoplay',
 	captionsCookieName: 'featuredVideoCaptions',
 	playerCookieExpireDays: 14,
-	placeholderImage: computed('model', function () {
-		return this.get('model.embed.jsParams.playlist.0.image');
-	}),
+	placeholderImage: readOnly('model.embed.jsParams.playlist.0.image'),
 	placeholderStyle: computed('placeholderImage', function () {
 		return htmlSafe(`background-image: url(${this.get('placeholderImage')})`);
 	}),
 
-	// when navigating from one article to another with video, we need to destroy player and
-	// reinitialize it as component itself is not destroyed. Could be done with didUpdateAttrs
-	// hook, however it is fired twice with new attributes.
-	videoIdObserver: on('didInsertElement', observer('model.embed.jsParams.videoId', function () {
+	didUpdateAttrs() {
```

Comment:
Isn't it fired twice?

Reply:
nope, I checked it and it fired only once